homeassistant/components/sensor/__init__.py

```diff
@@ -55,6 +64,13 @@
     DEVICE_CLASS_HUMIDITY,  # % of humidity in the air
     DEVICE_CLASS_ILLUMINANCE,  # current light level (lx/lm)
     DEVICE_CLASS_MONETARY,  # Amount of money (currency)
+    DEVICE_CLASS_OZONE,  # Amount of O3 (ppm)
+    DEVICE_CLASS_NITROGEN_DIOXIDE,  # Amount of NO2 (ppm)
+    DEVICE_CLASS_NITROGEN_OXIDE,  # Amount of NO (ppm)
+    DEVICE_CLASS_NITROGEN_MONOXIDE,  # Amount of N2O (ppm)
+    DEVICE_CLASS_PM1,  # Particulate matter <= 0.1 μm
```

Comment:
Do these also use ppm unit?

Reply:
Usually µg/m³.

Reply:
Yep, µg/m³. Will update the comments and check out the CI errors.

Reply:
Actually homekit specifies O3 and NO2 as densities in µg/m³, so I will dig a bit more into that. We should be able to convert between the 2 (https://cfpub.epa.gov/ncer_abstracts/index.cfm/fuseaction/display.files/fileid/14285). If @ahayworth is still interested in this he might have some input on what to do here as well.

Reply:
I am no expert in this field but I recently became very interested in this functionality in HA and started looking for a good source for my area. During my reading I saw that some countries use ppm, others ppb, and some µg/m³. 

To link it to this point, I read that the conversion between ppm and µg/m³ isn't straight forward, as density is a function of temperature and air pressure. I definitely got the impression that ppm is most commonly used for NO2, SO2 and O3 but not always. Btw, the link above also states:
> Also, atmospheric temperature and pressure affect the calculation.

Perhaps where conversions are done, some average assumptions would need to be made to keep it simple. I also think the units might be localised ([see](https://en.wikipedia.org/wiki/Air_quality_index) e.g. how Australia uses ppm for NO2 but the UK uses µg/m³).

Reply:
Have updated all of these to µg/m³ (aside from AQI, which is unitless).

This is after a review of all existing air_quality sensors.

* Where I could find documentation (6 out of 9) they used µg/m³.
* ampio seemed to be down so i couldn't check its API. 
* dyson is undocumented, but based on the measurement to AQI conversion happening in homebridge-dyson, it looks like µg/m³
* blebox has a well documented API, but don't give units. Again, based on their AQI conversion the values seem to be  µg/m³

In terms of re-exporting the sensors, Alexa and OK Google don't know about these classes. I don't know if Siri can act of them but Siri is aware of them via the HomeKit bridge. Just like homekit_controller, this is also µg/m³.

So I believe these are the correct units now.

Reply:
> If @ahayworth is still interested in this he might have some input on what to do here as well.

I am generally still interested, but I have no strong feelings on what to do here, unfortunately.

Reply:
I was worried that the device classes here would be totally inconsistent unit wise when I tagged you and wondered if you could remember anything from your own review. But from the one I have just done, I don't think there is anything to worry about for these classes and the values above are good. 

(There are some completely non standard device specific classes in there, but out of scope of this PR)